C4SGSvc/src/main/java/org/c4sg/controller/UserController.java

```diff
@@ -81,4 +87,60 @@ public void deleteUser(@ApiParam(value = "User id to delete", required = true)
                                 @RequestParam(required = false) String lastName) {
         return userService.search(userName, firstName, lastName);
     }
+    
+    @RequestMapping(value = "/{userId}/uploadAvatar", method = RequestMethod.POST)
+    @ApiOperation(value = "Add new upload Avatar")
+    public String uploadAvatar(@ApiParam(value = "user Id", required = true)
+                             @PathVariable Integer userId,
+                             @ApiParam(value = "Request Body", required = true)
+                             @RequestBody String avatarFileContent) {
```

Comment:
Don't we need a  `@RequestPart("file") MultipartFile avatarFileContent` , here ?